### PR TITLE
Add stack using array

### PR DESCRIPTION
**Pull request by muthuannamalai12**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Added Stack Implementation using Array in C.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#1982 
(Write your answer here.)

### CODE REVIEW

**Comment by muthuannamalai12** [comment]
@atarax665 Kindly Review

**Comment by muthuannamalai12** [comment]
@atarax665 Kindly Review

File: C/ds/StackImplementationWithArray.c [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+3.	Display
+4.	peek
+5.	exit
+Enter your choice: 3 
+Stack elements are: 45
+
+Time complexity : O(1)
+Space Complexity: O(n)
+*/
+
```
Comment (srimanikantapalakollu) [inline C/ds/StackImplementationWithArray.c]: @muthuannamalai12 Remove all the unnecessary extra lines at the end of the file.

File: C/ds/StackImplementationWithArray.c [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+1.push() − Pushing (storing) an element on the stack.
+2.pop() − Removing (accessing) an element from the stack.
+3.peek() − get the top data element of the stack, without removing it.
+4.display() - used to display the elements
+*/
+
+#include <stdio.h>
+#include <unistd.h>
+#include <stdlib.h>
+#define SIZE 10
```
Comment (srimanikantapalakollu) [inline C/ds/StackImplementationWithArray.c:13]: @muthuannamalai12 Try to assign the size of an array at runtime.

**Review by srimanikantapalakollu — CHANGES_REQUESTED** [review]

**Comment by muthuannamalai12** [comment]
I have changed it. Kindly review it @srimani-programmer

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**